### PR TITLE
Add pyaaf2 recipe

### PR DESCRIPTION
**Pull request by vvzen**
Hi all!
This is my first PR here, I hope got everything right but there was a lot to cover and read through!
Adding this package for reading .aaf files in Python since it's used as a dependency by https://github.com/PixarAnimationStudios/OpenTimelineIO , which is the main repo I wanted to build a package for.
Thanks!

Valerio

### CODE REVIEW

File: recipes/pyaaf2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 160d3c26c7cfef7176d0bdb0e55772156570435982c3abfa415e89639f76e71b
+
+build:
+  number: 0
```
Comment (wolfv) [inline recipes/pyaaf2/meta.yaml:13]: It looks like this package could be "noarch": https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-builds

just add `noarch: python`

File: recipes/pyaaf2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
```
Comment (wolfv) [inline recipes/pyaaf2/meta.yaml:21]: When you make this noarch, please specify a lower bound for python, e.g. `python >=3.6`

File: recipes/pyaaf2/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    - aaf2.model
+    - aaf2.model.ext
+
+about:
+  home: "https://github.com/markreidvfx/pyaaf2"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: "A python module for reading and writing advanced authoring format files"
+  doc_url:
```
Comment (wolfv) [inline recipes/pyaaf2/meta.yaml:35]: if you don't know a good value for this, just remove the keys

**Review by wolfv — CHANGES_REQUESTED** [review]

**Comment by wolfv** [comment]
great!

**Comment by vvzen** [comment]
Best PR experience ever! Thanks :D

**Pull request merged**